scripts/workers/submission_worker.py

```diff
@@ -446,20 +446,23 @@ def main():
     sys.path.append(COMPUTE_DIRECTORY_PATH)
 
     load_active_challenges()
-
     connection = pika.BlockingConnection(pika.ConnectionParameters(
-        host='localhost'))
+        host=RABBITMQ_PARAMETERS.get('HOST', None)))
 
     channel = connection.channel()
-    channel.exchange_declare(exchange='evalai_submissions', type='topic')
+    channel.exchange_declare(
+        exchange=RABBITMQ_PARAMETERS.get('EXCHANGE_NAME', None),
```

Comment:
I will prefer that you make key as `EVALAI_EXCHANGE` and then `NAME` and `TYPE` further to nest that dict.